libsolidity/interface/CompilerStack.h

```diff
@@ -349,6 +349,11 @@ class CompilerStack: public langutil::CharStreamProvider
 	/// caused by differences in metadata. Should only be used for testing.
```

Comment:
The comment here (starting in the line before) probably should be removed now that the method is used [here](https://github.com/ethereum/solidity/blob/75999e4f4c5171648bea880da7d2250bea338885/libsolidity/interface/StandardCompiler.cpp#L1093) and [here](https://github.com/ethereum/solidity/blob/75999e4f4c5171648bea880da7d2250bea338885/solc/CommandLineInterface.cpp#L692).

Reply:
Good catch. Going to change now.